tests/attr.py

```diff
@@ -0,0 +1,14 @@
+def attr(*args, **kwargs):
+    """Decorator that adds attributes.
+
+    """
+    def wrap(x):
+        for k in args:
+            setattr(x, k, True)
+        for k, v in kwargs.items():
+            setattr(x, k, v)
+        return x
+    return wrap
+
+gpu   = attr('gpu')
+cudnn = attr('gpu', 'cudnn')
```

Comment:
Move these decorators to somewhere in chainer package (e.g. by adding chainer.testing module) and avoid relative imports from the test codes.